### PR TITLE
Feature/solana api new methods

### PR DESCRIPTION
**Pull request by ginstagram**
## Description
Updates of Solana API reference and misc

add 5 new methods
describe limitations for Solana API
edit previously added methods for Solana API
change heading from Chainstack Marketplace to Marketplace
add release notes
change version

### CODE REVIEW

File: docs/api/solana/solana-api-reference.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ To read data from and send transactions to the Solana blockchain, an application
 
 When communicating with a Solana RPC node, the Solana client implements a JSON-RPC specification, a communication protocol allowing one to make remote calls and execute them as if they were made locally.
```
Comment (akegaviar) [inline docs/api/solana/solana-api-reference.md:27]: There's as typo in 

```
To read data from and send transactions to the Solana blockchain, an application must connect to an Solana RPC node.
```

File: docs/api/solana/getmultipleaccounts.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    content: getMultipleAccounts JSON-RPC method for the Solana API available with examples in Solana web3.js, Solana.py, and cURL.
+  - name: keywords
+    content: json rpc methods curl api solana.py solana web3.js javascript python solana
+---
+
+# getMultipleAccounts
+
+::: tip Information
+
+This method is currently disabled for Sonala devnet elastic nodes.
```
Comment (akegaviar) [inline docs/api/solana/getmultipleaccounts.md:13]: `Sonala`

File: docs/api/solana/getmultipleaccounts.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
+meta:
+  - name: description
+    content: getMultipleAccounts JSON-RPC method for the Solana API available with examples in Solana web3.js, Solana.py, and cURL.
+  - name: keywords
+    content: json rpc methods curl api solana.py solana web3.js javascript python solana
+---
+
+# getMultipleAccounts
```
Comment (akegaviar) [inline docs/api/solana/getmultipleaccounts.md:9]: Page not rendering. 
SCR-20221128-q0n

Comment (ginstagram) [reply docs/api/solana/getmultipleaccounts.md:9]: > It renders on https://docs.dev.chainstack.com/api/solana/getmultipleaccounts

File: docs/api/solana/getprogramaccounts.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    content: json rpc methods curl api solana.py solana web3.js javascript python solana
+---
+
+# getProgramAccounts
+
+Solana API method that returns all accounts owned by the provided program public key.
+
+## Temporary limitations
+
+This method is currently disabled for Sonala devnet elastic nodes.
```
Comment (akegaviar) [inline docs/api/solana/getprogramaccounts.md:15]: `Sonala`

File: docs/api/solana/gettokenaccountsbydelegate.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    content: getTokenAccountsByDelegate JSON-RPC method for the Solana API available with examples in Solana web3.js, Solana.py, and cURL.
+  - name: keywords
+    content: json rpc methods curl api solana.py solana web3.js javascript python solana
+---
+
+# getTokenAccountsByDelegate
+
+::: tip Information
+
+This method is currently disabled for Sonala devnet elastic nodes.
```
Comment (akegaviar) [inline docs/api/solana/gettokenaccountsbydelegate.md:13]: Fix all Sonalas please

**Review by akegaviar — CHANGES_REQUESTED** [review]
Changes please

**Review by soos3d — DISMISSED** [review]
LGTM

File: docs/api/solana/getblocks.md [hunk 6]
```diff
@@ -3,29 +3,29 @@ meta:
...
-    content: json rpc methods curl api solana.py solana web3.js javascript python solana 
+    content: json rpc methods curl api solana.py solana web3.js javascript python solana
 ---
 
 # getBlocks
 
-Solana API method that returns a list of confirmed blocks between two slots. 
+Solana API method that returns a list of confirmed blocks between two slots.
 
 ::: tip
```
Comment (akegaviar) [inline docs/api/solana/getblocks.md:13]: Make it consistent please with the other blocks here.

ie 

```
::: tip Information
```

File: docs/api/solana/getmultipleaccounts.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+::: tip Information
+
+This method is currently disabled for Solana devnet elastic nodes.
```
Comment (akegaviar) [inline docs/api/solana/getmultipleaccounts.md:13]: Here and everywhere

```
elastic Solana nodes on the devnet
```

or

```
elastic Solana devnet nodes
```

Guys, we have one strict order that needs to be consistent across all assets. Add it to our style guide on notion and commit to it please.

[Example](https://docs.chainstack.com/operations/ethereum/debug-and-trace-apis) of where it's implemented correctly: `elastic Ethereum archive node`.

If we add network to it, then it's `elastic Ethereum archive mainnet node` or `elastic Ethereum archive node on the mainnet`.

The order should be strict and it's [type](https://docs.chainstack.com/operations/ethereum/types) [blockchain](https://docs.chainstack.com/blockchains/ethereum) [mode](https://docs.chainstack.com/operations/ethereum/modes) [network](https://docs.chainstack.com/operations/ethereum/networks) node.

**Review by akegaviar — CHANGES_REQUESTED** [review]
Changes and make sure the order is in the style guide.

File: docs/api/solana/getmultipleaccounts.md (hunk 7 above)
Comment (ginstagram) [reply docs/api/solana/getmultipleaccounts.md:13]: Added to the content style guide

**Review by akegaviar — APPROVED** [review]

File: docs/api/solana/gettokenaccountsbyowner.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+:::
+
+Solana API method that returns all SPL token accounts by token owner.
+
+**Parameters:**
+
+* `<string>` — public key of the account owner to query, as a base-58 encoded string
+* `<object>` — one of the following:
+  * `mint: <string>` - the public key of the NFT collection, as a base-58 encoded string;
```
Comment (conor2e) [inline docs/api/solana/gettokenaccountsbyowner.md:23]: Check punctuation at the end of this line and the next.

File: docs/api/solana/accountsubscribe.md [hunk 9]
```diff
@@ -24,9 +24,9 @@ To use the Solana API subscriptions with cURL, use the code example as a message
...
-  * `encoding: <string>` — the encoding for account data, either `base58` (slow), `base64`, `base64+zstd` or `jsonParsed`. `jsonParsed` encoding attempts to use program-specific state parsers to return more human-readable and explicit account state data. If `jsonParsed` is requested, but a parser cannot be found, the field falls back to binary encoding, detectable when the data field is type `<string>`.
+* `<object>` — (optional) a configuration object containing the following optional fields:
+  * `commitment: <string>` — (optional) the commitment.
+  * `encoding: <string>` — the encoding for account data, either `base58` (slow), `base64`, `base64+zstd`, or `jsonParsed`. `base58` is limited to account data of less than 128 bytes. `base64` will return base64 encoded data for account data of any size. `base64+zstd` compresses the account data using Zstandard and base64 encodes the result. `jsonParsed` encoding attempts to use program-specific state parsers to return more human-readable and explicit account state data. If `jsonParsed` is requested, but a parser cannot be found, the field falls back to base64 encoding, detectable when the data field is `<string>` type.
```
Comment (conor2e) [inline docs/api/solana/accountsubscribe.md:29]: base64 should have hyphen in last sentence.

**Review by conor2e — CHANGES_REQUESTED** [review]
Some minor formatting checks required.

**Review by akegaviar — APPROVED** [review]

**Review by conor2e — APPROVED** [review]

**Pull request merged**